### PR TITLE
More style checking.

### PR DESCRIPTION
**Pull request by madmaxoft**
Spaces around some operators are checked.

### CODE REVIEW

**Comment by madmaxoft** [comment]
Unfortunately the spaces-around-operators is quite difficult and had a few false positives, so I had to work around them. Spaces around minus cannot be checked at all, due to minus being used as a hyphen between words.

File: src/Entities/Minecart.cpp [hunk 1]
```diff
@@ -954,7 +954,7 @@ bool cMinecart::TestEntityCollision(NIBBLETYPE a_RailMeta)
 			}
 
 			/* Check to which side the minecart is to be pushed.
-			Let's consider a z-x-coordinate system where the minecart is the center (0/0).
+			Let's consider a z-x-coordinate system where the minecart is the center (0 / 0).
```
Comment (bearbin) [inline src/Entities/Minecart.cpp]: This should be (0,0) anyway?

**Comment by bearbin** [comment]
There are a huge amount of false positives for "/", so many I don't think it's really worth it to check for spaces there.

**Comment by madmaxoft** [comment]
It's not that bad, and I worked around all of them anyway.

**Comment by tigerw** [comment]
Those are quite a lot of somewhat unnecessary changes; could you get the script to ignore lines that are comments?

**Comment by madmaxoft** [comment]
Oh I don't mind the changes in the comments at all, all of those are good. The only thing I don't quite like is the `" HTTP/1.0" needing to be split into two strings. If you can write a Lua pattern that matches the others but not this, be my guest.

**Comment by tigerw** [comment]
Though it's still an additional (comment) style for people to conform to, and not out of necessity either, but a side-effect from something else. Since comments should follow language, not code conventions, it would be better for the style checker to ignore them. @bearbin also recommends that similarly strings be ignored also, which also fixes your problem.

**Comment by worktycho** [comment]
The problem with comments is that its difficult to write a pattern to check is something is in a comment. I could probably manage a perl regex to do it, but even that would be a mess. Its certainly problematic with patterns to avoid: `/* a b c */ a/a /* a */ ` getting ignored.

**Comment by worktycho** [comment]
The regex is: `^(\/\*([^*]|\*[^/])*\*\/[^op]*)*(([^ ]op)|op[^ *])`  where op is the operator. I don't know how you'ld write that in lua patterns.

**Comment by madmaxoft** [comment]
I've fixed the operator-space patterns, now they ignore everything near doublequotes. So I got rid of the unwanted changes.

**Comment by worktycho** [comment]
This is being got by the broken master. Can you rebase on to the fix?

**Comment by madmaxoft** [comment]
The Appveyor build has been stuck so I cancelled it, the other builds passed.

**Pull request merged**